Source/Particles/Collision/MCCProcess.cpp

```diff
@@ -97,6 +97,9 @@ MCCProcess::readCrossSectionFile (
                                   amrex::Gpu::HostVector<amrex::Real>& sigmas )
 {
     std::ifstream infile(cross_section_file);
+    if(!infile) amrex::Abort("Failed to open readCrossSectionFile file");
```

Comment:
Based on the description in [this](https://gehrcke.de/2011/06/reading-files-in-c-using-ifstream-dealing-correctly-with-badbit-failbit-eofbit-and-perror/) article should we maybe change this whole `readCrossSectionFile` function to:
```
    std::ifstream infile(cross_section_file);
    if(!infile.is_open()) amrex::Abort("Failed to open cross-section data file");

    double energy, sigma;
    while (infile >> energy >> sigma) {
        energies.push_back(energy);
        sigmas.push_back(sigma);
    }
    if (infile.bad()) amrex::Abort("Failed to read cross-section data from file.");
    infile.close();
```

Reply:
That looks good to me as well :)